src/main/java/uk/gov/companieshouse/controller/AppealController.java

```diff
@@ -54,6 +59,12 @@ public AppealController(AppealService appealService) {
             return ResponseEntity.status(HttpStatus.UNAUTHORIZED).build();
         }
 
+        String validationError = appealReasonValidator.validate(appeal.getReason());
```

Comment:
If we were doing this in a scalable way we'd probably want to return a collection, incase a new field that needs to be validated by this as added.

Reply:
We did think of this but decided to keep it simple and only implement what's needed now. The ResponseEntity takes a String so it was easier to use that than having to modify the controller method's return type.

Reply:
Being left to a later refactoring ticket.